clients/cloud/python/README.md

```diff
@@ -103,13 +106,14 @@ Waiting for message or event/error in poll()
 ...
 ```
 
-
 # Example 2: Avro And Confluent Cloud Schema Registry
 
 This example is similar to the previous example, except the key and value are formatted as Avro and integrates with the Confluent Cloud Schema Registry.
 Before using Confluent Cloud Schema Registry, check its [availability and limits](https://docs.confluent.io/current/cloud/limits.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud).
 Note that your VPC must be able to connect to the Confluent Cloud Schema Registry public internet endpoint.
 
+These examples utilize the latest Serializer API provided by the `confluent-kafka` library.  The Serializer API replaces the legacy AvroProducer and AvroConsumer classes to provide a more flexible API including additional support for JSON, Protobuf, and Avro data formats.  See the latest [confluent-kafka documentation](https://docs.confluent.io/current/clients/confluent-kafka-python/#) for further details.
```

Comment:
the new apis are 'experimental and subject to change', is the main thing. In practice, they're essentially stable, except that I hope they'll be combined with the Producer and Consumer classes (not exposed as separate classes), and the details of that aren't completely clear.

i'm not sure how best to communicate this.

Reply:
Probably don't want/need the `#` in the url.